tests/sentry/integrations/vercel/test_uninstall.py

```diff
@@ -23,6 +23,50 @@
 
 
 class VercelUninstallTest(APITestCase):
+    def setUp(self):
+        self.url = "/extensions/vercel/delete/"
+        metadata = {
+            "access_token": "my_access_token",
+            "installation_id": "my_config_id",
+            "installation_type": "team",
+            "webhook_id": "my_webhook_id",
+        }
+        self.integration = Integration.objects.create(
+            provider="vercel",
+            external_id="vercel_team_id",
+            name="My Vercel Team",
+            metadata=metadata,
+        )
+        self.integration.add_organization(self.organization)
+
+    def _get_delete_response(self):
+        # https://vercel.com/docs/integrations?query=event%20paylo#webhooks/events/integration-configuration-removed
+        return """{
+            "payload": {
+                "configuration": {
+                    "id": "my_config_id",
+                    "projects": ["project_id1"]
+                }
+            },
+            "teamId": "vercel_team_id",
+            "userId": "vercel_user_id"
+        }"""
+
+    def test_uninstall(self):
+        response = self.client.post(
```

Comment:
This can be replaced with the `get_success_response` helper:
```        
self.get_success_response(**self._get_delete_response())
```
which automatically sets the `content_type` asserts the `status_code`.

To use it just add `endpoint` and `method` to the test class:
```
class VercelUninstallTest(APITestCase):
    endpoint = "sentry-extensions-vercel-delete"
    method = "post"
```
and make `_get_delete_response` return a dict.

Reply:
nice ! since tests are passing and this kind of blocks my other PR im going to merge and then I'll update those tests with this